recipe-box-backend/src/main/java/com/endava/recipebox/response/CustomRecipeMapper.java

```diff
@@ -0,0 +1,12 @@
+package com.endava.recipebox.response;
+
+import com.endava.recipebox.model.Recipe;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper
+public interface CustomRecipeMapper {
```

Comment:
Inside the mapper we can have multiple mappings between different type of wrappers created for Recipes. This is why I prefer to be names as RecipeMapper